modules/ossm-installing-multi-primary-multi-network-kiali.adoc

```diff
@@ -0,0 +1,216 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-multi-primary-multi-network-kiali_{context}"]
+= Installing a multi-primary multi-network mesh by using Kiali
+
+Install {istio} in the multi-primary multi-network topology on two {ocp-product-title} clusters.
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster.
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the latest Kiali Operator on each cluster.
+
+* Istio installed in a multi-cluster configuration on each cluster.
+
+* You have `istioctl` installed on the laptop you can use to run these instructions.
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+
+* You have configured a metrics store so that Kiali can query metrics from all the clusters. Kiali queries metrics and traces from their respective endpoints.
+
+.Procedure
+
+. Install Kiali on the East cluster:
+
+.. Create a YAML file named `kiali.yaml` that creates a namespace for the Kiali deployment. 
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: istio-system
+spec:
+  version: default
+  external_services:
+    prometheus:
+      auth:
+        type: bearer
+        use_kiali_token: true
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+----
++
+[NOTE]
+====
+The endpoint for this example uses OpenShift Monitoring to configure metrics.
```

Comment:
Is it possible to link to these docs for configuring openshift monitoring? https://96498--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/observability/kiali/ossm-kiali#ossm-config-openshift-monitoring-kiali_ossm-kiali